x-pack/legacy/plugins/siem/public/pages/hosts/details/index.tsx

```diff
@@ -57,11 +57,39 @@ const HostDetailsComponent = React.memo<HostDetailsComponentProps>(
     return (
       <>
         <WithSource sourceId="default">
-          {({ indicesExist, indexPattern }) =>
-            indicesExistOrDataTemporarilyUnavailable(indicesExist) ? (
+          {({ indicesExist, indexPattern }) => {
+            const filterQuery = convertToBuildEsQuery({
+              indexPattern,
+              queries: [query],
+              filters: [
+                {
+                  meta: {
+                    alias: null,
+                    negate: false,
+                    disabled: false,
+                    type: 'phrase',
+                    key: 'host.name',
+                    value: detailName,
+                    params: {
+                      query: detailName,
+                    },
+                  },
+                  query: {
+                    match: {
+                      'host.name': {
+                        query: detailName,
+                        type: 'phrase',
+                      },
+                    },
```

Comment:
Very cool! This refactor has really simplified implementation details like narrowing to a specific filter like this.